docs/usage/estimating-moi.md

```diff
@@ -35,5 +24,36 @@ However, this would likely be an underestimate because most FRC robots tend to h
 
 A better estimate could by found by summing each subsystem's contributions to the robot's overall MoI, based on its mass and average distance from the axis of rotation:
 
-$$ I = \sum_{i=1}^n \text{mass}_\text{subsystem i} * \text{radius}_\text{subsystem i}^2 $$
+$$ I = \sum_{i=1}^n \text{mass}\_\text{subsystem i} * \text{radius}\_\text{subsystem i}^2 $$
```

Comment:
Don't escape the underscore. It fixes the GitHub rendering but breaks the MkDocs MathJax rendering (so the published website won't render the math correctly).